### PR TITLE
 [Fused] LeakyRelu op

### PR DESCRIPTION
**Pull request by postrational**

### CODE REVIEW

File: src/ngraph/op/fused/leaky_relu.cpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+{
+    constructor_validate_and_infer_types();
+}
+
+NodeVector op::LeakyRelu::decompose_op() const
+{
+    auto data = get_argument(0);
+    auto alpha_node = get_argument(1);
+
+    alpha_node = ngraph::op::make_broadcast_node(alpha_node, data->get_shape());
```
Comment (tsocha) [inline src/ngraph/op/fused/leaky_relu.cpp]: Broadcasting `[2, 1]` with `[1, 3]` will fail with this broadcasting.
Consider use of numpy style broadcasting.

File: test/backend_fused_op.in.cpp [hunk 2]
```diff
@@ -716,6 +716,42 @@ NGRAPH_TEST(${BACKEND_NAME}, mvn_mean_variance_normalization_split_channels)
...
+NGRAPH_TEST(${BACKEND_NAME}, leaky_relu)
+{
+    auto data_node = make_shared<op::Parameter>(element::f64, Shape{3, 4});
+    auto alpha_node = make_shared<ngraph::op::Constant>(element::f64, Shape{}, vector<double>{0.1});
+    auto leaky_relu = make_shared<op::LeakyRelu>(data_node, alpha_node);
+
+    auto function = make_shared<Function>(NodeVector{leaky_relu}, ParameterVector{data_node});
+    auto test_case = ngraph::test::NgraphTestCase(function, "${BACKEND_NAME}");
+
+    test_case.add_input<double>({std::numeric_limits<double>::max(),
```
Comment (arogowie-intel) [inline test/backend_fused_op.in.cpp]: `std::` prefix is unnecessary.

File: src/ngraph/frontend/onnx_import/op/leaky_relu.cpp [hunk 3]
```diff
@@ -16,19 +16,13 @@
 
 #include <memory>
 
+#include "core/node.hpp"
 #include "exceptions.hpp"
```
[the remark this answers is not in the document]
Comment (postrational) [reply src/ngraph/frontend/onnx_import/op/leaky_relu.cpp]: It's in use.

File: src/ngraph/runtime/intelgpu/intelgpu_backend.cpp [hunk 4]
```diff
@@ -2174,6 +2176,8 @@ bool runtime::intelgpu::IntelGPUBackend::is_supported_impl(const Node& node)
     case OP_TYPEID::Elu:
     case OP_TYPEID::Gemm:
     case OP_TYPEID::GRN:
+    case OP_TYPEID::GRN:
```
Comment (dmyershov) [inline src/ngraph/runtime/intelgpu/intelgpu_backend.cpp]: GRN is mentioned twice here.

**Comment by postrational** [comment]
TESTNOW

**Comment by aslepko** [comment]
TESTNOW

**Pull request merged**